server.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express")
+const app = express();
+
+
+app.listen(3000,()=>{
```

Comment:
It's a good practice to make the server port configurable. You can use the `process.env.PORT` environment variable, with a default value of 3000. This way, you can easily change the port in different environments. [medium]